eventuate-local-java-cdc-connector-common/src/main/java/io/eventuate/local/common/MessageCleanerPurgeProperties.java

```diff
@@ -0,0 +1,62 @@
+package io.eventuate.local.common;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import org.apache.commons.lang.builder.ReflectionToStringBuilder;
+import org.springframework.util.Assert;
+
+import java.util.concurrent.TimeUnit;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class MessageCleanerPurgeProperties{
+  private boolean purgeMessagesEnabled = false;
```

Comment:
This could be `messagesEnabled` ?

Reply:
yes, but it makes sense only in context. I wanted to invent something that does not repeat 'purge' but also makes sense without outer object. But if you think that is fine, I will just remove 'purge'.